app/controllers/games_controller.rb

```diff
@@ -13,7 +13,12 @@ def create
 
   def show
     @game = Game.find_by_id(params[:id])
-    render text: 'Not Found', status: :not_found if @game.blank?
```

Comment:
Just add return to this line.
`return` render text: 'Not Found', status: :not_found if @game.blank?`